doc/users/colors.rst

```diff
@@ -14,58 +14,9 @@ In almost all places in matplotlib where a color can be specified by the user it
 * valid css4/X11 color names
 * valid name from the `XKCD color survey
   <http://blog.xkcd.com/2010/05/03/color-survey-results/>`__ These
-  names are available both with and with out spaces.  In the case of name clashes
-  the css/X11 names have priority.  To ensure colors
-  from the XKCD mapping are used prefix the space-less name with
-  ``'XKCD'``.
+  names are prefixed with ``'xkcd:'`` (e.g., ``'xkcd:sky blue'``) to
+  prevent name clashes with the CSS4/X11 names.
 
 All string specifications of color are case-insensitive.
 
 Internally, mpl is moving to storing all colors as RGBA float quadruples.
-
-Name clash between CSS4/X11 and XKCD
```

Comment:
Even if we are explicitly name-spacing the colors, I think this chart is still interesting to include in the docs.